src/indexApi.ts

```diff
@@ -16,12 +23,24 @@ const asAnalyticsReq = asObject({
   timePeriod: asString
 })
 
-const asCheckTxReq = asObject({
+const asCheckTxsReq = asObject({
   apiKey: asString,
-  pluginId: asString,
-  orderId: asString
+  data: asArray(
+    asObject({
+      pluginId: asString,
+      orderId: asString
+    })
+  )
 })
 
+const asCheckTxsServerResponse = asArray(
```

Comment:
This really should be `asTxFetch`. This is not a server response but rather the response of the database.